Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12-alpine AS builder
+FROM node:14-alpine AS builder
```

Comment:
[Node 12 is LTS until May 2022](https://nodejs.org/en/about/releases/) but we might as well upgrade

Reply:
Actually ended up going to node 16, see [comment below](https://github.com/christchurchlondon/foodbank-southlondon/pull/95#issuecomment-925242177)

Reply:
Fab, does this line need updating to 16 then? Also, I think we mention npm version in readme - can you update the text/link there?

Reply:
Yep have done in [this commit](https://github.com/christchurchlondon/foodbank-southlondon/pull/95/commits/5c06e11d12261cb788dc6f36abf012994bc8014b). I think I forgot to push it last night 🤦